core/src/main/java/io/grpc/Metadata.java

```diff
@@ -170,6 +170,13 @@ private void storeAdd(String name, MetadataEntry value) {
   }
 
   /**
+   * Returns the total number of headers in this metadata, including duplicates.
```

Comment:
This probably should be a bit more precise, since "size" is a bit ambiguous. It should be clear this refers to the number of values (or equivalent)